lib/publiccloud/azure_client.pm

```diff
@@ -62,6 +64,45 @@ sub vault_create_credentials {
     }
 }
 
+=head2 configure_docker
+
+Configure the docker to access the cloud provider registry
+=cut
+
+sub configure_docker {
```

Comment:
```suggestion
sub registry_auth {
```
or something else , but `configure_docker` does not match for sure because only thing what you are doing here is auth

Reply:
yes, but in others 2 cases we call that configure_docker... we can change this one to registry_auth. But for the same reason we should rename the others. (but not in this PR)

Reply:
in some way in all cases it modifies the configuration for docker config to add the repository

Reply:
ok , if this is to keep it aligned with other implementations I am fine with that